sql_generators/glean_usage/templates/app_ping_view.view.sql

```diff
@@ -10,8 +10,12 @@ SELECT
   "{{ query.dataset }}" AS normalized_app_id,
   {% if query.channel and query.app_name != "fenix" -%}
   "{{ query.channel }}" AS normalized_channel,
-  {% elif query.app_name == "fenix" %}
+  {% elif query.app_name == "fenix" and query.includes_client_info %}
   mozfun.norm.fenix_app_info("{{ query.dataset }}", client_info.app_build).channel AS normalized_channel,
+  {% elif query.app_name == "fenix" and not query.includes_client_info %}
+    -- set app build to 21850000 since all affected pings are coming from versions older than that
```

Comment:
Should this comment describe the issue? "affected pings" may not be clear to future us (though looking through git logs may be good enough?).